### ISSUE TITLE
v-row and v-col meaning is swapped

### ISSUE THREAD

**Issue by mfferreira** [vuetifyjs/vuetify#51]
It seems the v-col should be v-row and vice-versa.

Looking at the [docs source for grids](https://github.com/vuetifyjs/docs/blob/master/src/views/GridView.vue) there's only one v-row, when there's actually 6, and 6 v-col when there's only 1.

**Comment by johnleider** [comment]
I guess I'm not sure how you are drawing that conclusion. Could you explain further? I'm not the first come up with this grid naming structure, it's pretty well used everywhere.

**Comment by mfferreira** [comment]
In the documentation example the structure is the following:

    v-row
        v-col
        v-col
        v-col
        v-col
        v-col
        v-col

Which should mean that there's one row with 6 columns (let's ignore the for-loops).

But in reality, I think the markup should be:

    v-col
        v-row
            v-col
        v-row
            v-col
        v-row
            v-col
        v-row
            v-col
        v-row
            v-col
        v-row
            v-col

Basically what I'm saying is that the markup is telling me there's only one row, when there's really 6.
Unless I'm totally confusing something here of course.

**Comment by johnleider** [comment]
The columns are dynamic and part of a flex box. In order for that to work, they must be contained within the same row.

**Comment by mfferreira** [comment]
Right, that's what I found out when trying to add a few v-row's, which didn't make sense to me.

So if v-row doesn't really add new rows because v-col does it automagically, what's it purpose?

**Comment by Pwntus** [comment]
@mfferreira I guess you're not taking into consideration that when the total width of the cols exceed the row-width they will break line. This is perfectly normal. https://jsfiddle.net/5ohybqoe/

**Comment by mfferreira** [comment]
@Pwntus I totally understand that.

But actually I got it working as expected now. I'm not sure what I was doing wrong...
Closing.
